Kickstarter-iOS/Views/Controllers/ManagePledgeViewController.swift

```diff
@@ -0,0 +1,77 @@
+import KsApi
+import Library
+import Prelude
+import UIKit
+
+final class ManagePledgeViewController: UIViewController {
+  private lazy var closeButton: UIBarButtonItem = {
+    UIBarButtonItem(
+      image: UIImage(named: "icon--cross"),
+      style: .plain,
+      target: self,
+      action: #selector(RewardsCollectionViewController.closeButtonTapped)
+    )
+  }()
+
+  private lazy var editButton: UIBarButtonItem = {
+    UIBarButtonItem(
+      barButtonSystemItem: .edit,
+      target: self,
+      action: #selector(ManagePledgeViewController.editButtonTapped)
+    )
+  }()
+
+  private lazy var navigationBarShadowImage: UIImage? = {
```

Comment:
Are we using this at all? Looks like we're setting `shadowImage` to new `UIImage` in `viewDidLoad`?

Reply:
This will be used to show/hide the bar shadow depending on the scrolling position (hidden if it's scrolled to the top, shown otherwise).